### DISCUSSION TITLE
Quillnote Sync Server

### DISCUSSION THREAD

**Discussion by arunk140** [quillpad/quillpad#33] [Show and tell]
Use the Sync feature of Quillnote/Quillpad app without a full Nextcloud Instance. This server emulates all the required APIs for "Notes" from NextCloud.

Currently the notes data is stored in an sqlite db. 

Currently in Development - Adding support for other DBs and some more configuration options soon.

https://github.com/arunk140/quillnote-server

**Comment by jsixface** [comment]
This is cool, being written in Go you only need a tiny binary to run the server backend instead of the entire Nextcloud. Are you planning to write a web/other fronend for this ?

**Reply by arunk140** [reply]
Yep - planning a React/Next.js based frontend following a similar design style to Quillpad/Google Keep..

**Reply by jsixface** [reply]
Cool. Sounds good. 
Just FYI - I updated the README yesterday to reflect the [roadmap for this app](https://github.com/quillpad/quillpad#roadmap).
